Google.PowerShell/BigQuery/GcbqProject.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Google.Apis.Bigquery.v2;
+using Google.Apis.Bigquery.v2.Data;
+using System.Management.Automation;
+using System;
+
+namespace Google.PowerShell.BigQuery
+{
+    /// <summary>
+    /// <para type="synopsis">
+    /// Lists all projects to which you have been granted any project role.
+    /// </para>
+    /// <para type="description">
+    /// Queries to find which projects you have been granted any project role in and returns them in the form of ProjectsData objects.
+    /// The flag -MaxResults can be used to specify the maximum number of results retrieved at once, and -PageToken can be used to
+    /// specify an offset into the project list at which to start.
+    /// </para>
+    /// <example>
+    ///   <code>PS C:\> Get-GcbqProjects -MaxResults 20</code>
+    ///   <para>This command will list up to the first 20 BigQuery projects that you've been granted any roles for.</para>
```

Comment:
I believe these are Google Cloud Console projects.  I don't think BigQuery has the notion itself of projects.